oraichain/assetlist.json

```diff
@@ -533,6 +685,28 @@
       "name": "Btc",
       "display": "btc",
       "symbol": "BTC",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "bitcoin",
+            "base_denom": "btc"
+          },
+          "provider": "OraiBtc Bridge"
+        },
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "OraiBtc Bridge",
```

Comment:
is there a chain called OraiBTC Bridge?
If so, we'll need to register that

Reply:
OraiBTC Bridge is the chain to transfer funds from Oraichain to Bitcoin ( Oraichain -> OraiBTC -> Bitcoin )
Can i register that to the next pull request?

Reply:
Unfortunately we can't accept a trace like this. The chain_name being non-existence would surely break things.

Reply:
We also cannot define an ibc transfer that isn't registered(, and registering it requires the chain to be registered).
It's also problematic that an IBC transfer doesn't result in a denom starting with "ibc/"

Reply:
so it is not appropriate to describe the "traces" field here. I will hide it and add it when we add the Oraibridge network. 

thanks!!!

Reply:
Ideally we register the chains and then the ibc connections before registering these assets at all. But I don't want to hold up progress too much.